data/csp/oci/sle15/overlayfiles.yaml

```diff
@@ -0,0 +1,7 @@
+archive:
+  _namespace_oci_base:
+    _include_overlays:
+      - module-iscsi
+      - cloud-cfg-oci
+      - chrony-oci-ntp
```

Comment:
`chrony-oci-ntp` doesn't exist in `overlayfiles`. I think it did at some point but I removed it as it wasn't used.